src/index.ts

```diff
@@ -224,4 +226,17 @@ function listTargetVersions() {
     console.info(targetVersions.map((version) => `- ${version}`).join("\n"));
 }
 
+const originalError = console.error;
+const originalWarn = console.warn;
+
+console.error = (...args) => {
+    const formattedArgs = args.map((arg) => (typeof arg === "object" ? util.inspect(arg, { depth: null, colors: true }) : chalk.red(arg)));
```

Comment:
Otherwise error objects aren't displayed correctly

Before:
<img width="622" alt="Bildschirmfoto 2025-02-28 um 13 35 42" src="https://github.com/user-attachments/assets/52585500-9e1e-47b8-9297-e014e02ccf94" />



Now:
<img width="614" alt="Bildschirmfoto 2025-02-28 um 13 35 58" src="https://github.com/user-attachments/assets/a3e3d678-0c2b-48e0-9943-b74a9217411d" />

Reply:
Shouldn't we highlight objects as well?

Reply:
I'm not sure about that. I think making the whole object red makes it hard to read. That's at least my personal preference